client-misk/src/main/kotlin/app/cash/backfila/client/misk/internal/BackfillaParameters.kt

```diff
@@ -0,0 +1,87 @@
+package app.cash.backfila.client.misk.internal
+
+import app.cash.backfila.client.misk.Backfill
+import app.cash.backfila.client.misk.BackfillConfig
+import app.cash.backfila.client.misk.internal.BackfilaParametersOperator.Companion.TYPE_CONVERTERS
+import app.cash.backfila.protos.service.Parameter
+import com.google.inject.TypeLiteral
+import com.squareup.moshi.Types
+import java.lang.reflect.ParameterizedType
+import kotlin.reflect.KClass
+import kotlin.reflect.KParameter
+import kotlin.reflect.full.memberProperties
+import kotlin.reflect.full.primaryConstructor
+import kotlin.reflect.jvm.jvmErasure
+import okio.ByteString
+import okio.ByteString.Companion.encodeUtf8
+
+internal inline fun <reified T : Any> backfilaParametersForBackfill(backfillClass: KClass<Backfill<*, *, T>>): List<Parameter> {
```

Comment:
not a huge fan of top level scope functions that you use in other files, I would throw them into the class below as a companion, but up to you

Reply:
This happens at service startup. I wouldn't have a operator yet. I guess I could make one for each backfill at startup. Is that really better?

Reply:
I mean in the companion object, so it's static

Reply:
I can't get parametersToBytes in the companion object because it has to be inline... I'll merge it in as it is. If you have suggestions, LMK.